test/browser/support/F2F_CRI_START_SCHEMA.json

```diff
@@ -35,14 +35,34 @@
         },
         "component_id": {
             "type": "string"
+        },
+        "restricted": {
+            "type": "object",
+            "properties": {
+              "device_information": {
+                "type": "object",
+                "properties": {
+                  "encoded": {
+                    "type": "string"
+                  }
+                },
+                "required": [
+                  "encoded"
+                ]
+              }
+            },
+            "required": [
+              "device_information"
+            ]
         }
     },
     "required": [
         "event_name",
         "user",
         "timestamp",
         "event_timestamp_ms",
-        "component_id"
+        "component_id",
+        "restricted"
```

Comment:
Think we were going to say this wasn't required, right?

Reply:
Sorry not clear what you mean? I thought this ticket was to update the e2e tests so that they pass when the cloudfront function is deployed?